upup/pkg/fi/cloudup/bootstrapchannelbuilder.go

```diff
@@ -423,3 +462,141 @@ func (b *BootstrapChannelBuilder) buildManifest() (*channelsapi.Addons, map[stri
 
 	return addons, manifests, nil
 }
+
+func buildSecret() (kube_api.Secret, error) {
```

Comment:
This should be a

kops create secret

command

Rather than having the code here